va/va.h

```diff
@@ -2351,6 +2352,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
The two flags are added for slice decoding base mode (VA_DEC_SLICE_MODE_BASE). It would be better to add some comments here.